static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -20,6 +24,54 @@ type State = AsyncView['state'] & {
   inviteRequests: Member[];
 };
 
+function DisabledInviteMembersTooltip({children}: any) {
+  const title = tct(
+    `Enable this feature on your sentry installation by adding the
+    following configuration into your [configFile:sentry.conf.py].
+    See [configLink:the configuration documentation] for more
+    details.`,
```

Comment:
whoop, this needs some love

Reply:
Doesn't it say this by default somewhere in the disabled feature component?

Reply:
https://github.com/getsentry/sentry/blob/c51404d69b08f7e249edb9a480b039754bcace72/static/app/components/acl/featureDisabled.tsx#L106-L117

you should be able to just use the `FeatureDisabled` component

Reply:
Yeah, `<FeatureDisabled>` is working good imo. This is my fav layout so far:
<img width="1361" alt="Screen Shot 2022-03-22 at 1 37 00 PM" src="https://user-images.githubusercontent.com/187460/159571961-73c0c337-5b15-4def-8183-87ca129591e3.png">
<img width="1359" alt="Screen Shot 2022-03-22 at 1 37 08 PM" src="https://user-images.githubusercontent.com/187460/159571977-4b841581-d15c-4021-bb94-e7afa10c011a.png">

Earlier I was thinking that `<FeatureDisabled>` is too big and chunky in that part of the page. I was playing around, copy pasting, and came up with this kind of thing which I don't love:
<img width="476" alt="Screen Shot 2022-03-22 at 11 59 31 AM" src="https://user-images.githubusercontent.com/187460/159571622-5e225f2f-84cf-4d21-a747-8e027b3db13f.png">

Reply:
You can use the hovercard variant which might be more what you want

Reply:
Would it be crazy to copy+aste a bunch of the stuff from FeatureDisabled and put it inside a Hovercard? Could be a new reusable thing along the lines of `<FeatureDisableHovercard>`

Reply:
Yes it would, because you can just do this :) https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/wizard/index.tsx#L91-L104